src/styles/utilits/_normalize.scss

```diff
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+ul,
+li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
```

Comment:
here, you combine both resetting and normalizing styles, but shouldn't

Reply:
Can i leave it here for this task? or better remove?